src/diagnostics.ts

```diff
@@ -67,6 +67,12 @@ export class Range {
     );
   }
 
+  extendBy(delta: i32): Range {
```

Comment:
Perhaps `expandEnd`? If we ever needed it, this would also fit well with an `expandStart`.